src/components/Navbar.js

```diff
@@ -64,6 +64,7 @@ export class Navbar extends Component {
                       type="submit"
                       className="dropdown-item nav-link"
                       onClick={this.logoutUser}
+                      id="logout-user"
```

Comment:
Is this also about testing the untested areas?

Reply:
Yes, am supplying the button with a unique identifier in order to test it.